lib/slow_query_logger.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'digest'
+require 'logger'
```

Comment:
Presumably these are needed here because this class is loaded via an initializer which is before all the app's bundled gems are loaded?

Reply:
Correct.

Reply:
again - this question in the review may imply a comment is warranted (or not).